apps/docusaurus/docs/move/compiler_v2.md

```diff
@@ -2,13 +2,20 @@
 title: "Compiler v2 (beta)"
 ---
 
-# Move Compiler v2 (beta)
+# Move On Aptos Compiler (beta)
```

Comment:
Do we need the entire `Move on Aptos` in the name? I think it will be assumed it's for Aptos for most folks reading the Aptos docs but perhaps I'm missing context here.